Automattic/wp-calypso
# Design Picker: Add whitelist of standard themes v13n

#### Proposed Changes

* According to the pbxlJb-2gw-p2#comment-1692, we want to ship the verticalized standard themes from our top candidates
  * Geologist
  * Stewart
  * Arbutus
  * Zoologist 

Note that we don't pick just select Alonso, Bennett, and Calvin as we thought the images shown in those themes are still highly related to the copy.

https://user-images.githubusercontent.com/13596067/185352889-320fba45-eaec-4777-9f1d-fa614aee5978.mov

#### Testing Instructions



* Go to `/setup/designSetup?siteSlug=<your_site>`
* Check the following themes are verticalized
  * Geologist
  * Stewart
  * Arbutus
  * Zoologist 
* Continue with any of them and finish the onboarding flow
* Go to your home page and check whether it's verticalized or not

#### Pre-merge Checklist






Related to 77-gh-Automattic/ganon-issues, pbxlJb-2gw-p2#comment-1692, p1660126145013109/1660126031.069469-slack-CRWCHQGUB
